src/Data/Shortcut/Key.purs

```diff
@@ -0,0 +1 @@
+../../../generator/src/Data/Shortcut/Key.purs
```

Comment:
These are link files to files in generator/src. Either pulp or psc doesn't want to follow directory links but will follow individual file links.